pkgs/shells/bash/pgrp-pipe.patch

```diff
@@ -0,0 +1,24 @@
+Common subdirectories: ./CWRU and ../bash-4.4-fixed/CWRU
+Common subdirectories: ./builtins and ../bash-4.4-fixed/builtins
+diff -u ./configure.ac ../bash-4.4-fixed/configure.ac
+--- ./configure.ac	2016-09-07 22:56:28.000000000 +0200
++++ ../bash-4.4-fixed/configure.ac	2020-01-07 09:35:59.161167922 +0100
+@@ -1093,7 +1093,8 @@
+ lynxos*)	LOCAL_CFLAGS=-DRECYCLES_PIDS ;;
+ linux*)		LOCAL_LDFLAGS=-rdynamic		 # allow dynamic loading
+ 		case "`uname -r`" in
```

Comment:
This is impure. Better enable this unconditionally. Could use the patch from https://lists.defectivebydesign.org/archive/html/bug-bash/2016-06/msg00059.html

Reply:
Good one! (that patch had some more things in there so I created a similar one independently, both for 4.4 and 5.0)